testsuite/integration/microprofile-tck/health/src/test/resources/arquillian.xml

```diff
@@ -5,7 +5,7 @@
     <container qualifier="jboss" default="true" >
         <configuration>
             <property name="jbossHome">${jboss.home}</property>
-            <property name="javaVmArguments">${microprofile.jvm.args}</property>
+            <property name="javaVmArguments">${microprofile.jvm.args} -D__MP_HEALTH_TCK_DISABLE_SERVER_CHECKS=true</property>
```

Comment:
Why is it important to disable it? Was new output breaking MP TCK?

Reply:
The TCK must be run with any vendor probes disabled: https://github.com/eclipse/microprofile-health/blob/master/tck/running_the_tck.asciidoc#disabling-default-vendor-procedures

Reply:
fixed